IP_POLICY_GUIDANCE.md

```diff
@@ -67,8 +67,8 @@ and a [corporate CLA][CCLA].
 Projects can choose to adopt either the individual CLA by itself or
 both the individual CLA and the corporate CLA without any further review.
 
-If you need to use a different CLA, you will need to
-[obtaining an exemption from the board](#obtaining-an-exemption-from-the-board).
+If you need to use a different CLA, you must
+[obtain an exemption from the board](#obtaining-an-exemption-from-the-board).
```

Comment:
nit:
```suggestion
If you need to use a different CLA, you must [obtain an exemption from the board](#obtaining-an-exemption-from-the-board).
```
hard wrapping makes for much noisier diffs

Reply:
This doc uses [semantic line breaks](https://sembr.org/). Whether that's a good idea or not is debatable, but let's stay consistent in this PR. Happy to +1 a separate PR that changes the whole doc if people prefer a different style.

Reply:
That says "A semantic line break may occur before and after a hyperlink." - note "may" and not "must" :-) but sure, a nit is a nonblocker

Reply:
Point taken!

Reply:
(Also I can’t believe you actually read this.)